Tests/Core/AWSClientRuntimeTests/Auth/CredentialsProvidersTests/SSOCredentialsProviderTests.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import ClientRuntime
+import Foundation
+import XCTest
+
+@_spi(FileBasedConfig) @testable import AWSClientRuntime
+
+class SSOCredentialsProviderTests: XCTestCase {
+    let configPath = Bundle.module.path(forResource: "sso_tests", ofType: nil)!
+    let credentialsPath = Bundle.module.path(forResource: "credentials", ofType: nil)!
+    
+    func testCreateCredentialsProviderSSOLegacyProfile() async throws {
+        do {
+            let provider = try SSOCredentialsProvider(
+                profileName: "user",
+                configFilePath: configPath,
+                credentialsFilePath: credentialsPath)
+            XCTAssertNotNil(provider)
+            // get credentials will fail in CI due to expired token, so do not assert on credentials.
+            _ = try? await provider.getCredentials()
+        }
```

Comment:
I believe you can remove the do {} block, remove the XCTAssertNotNil and remove the ? from try?. 
- do {} block isnt being caught so doesn't seem necessary
- XCTAssertNotNil isn't necessary since if it results in an exception test will fail automatically
- try? is not necessary since we're not using the output and if its null we'll get an exception causing the test to fail